lib/components/easy-search-components.js

```diff
@@ -646,6 +646,32 @@ EasySearch.getComponentInstance = (function () {
 
         // Clear the selected items
         SearchVars.set(this.generateId(), { 'autosuggestSelected' : [] });
+      },
+      addToAutosuggestSelected : function(item, removeDuplicates) {
```

Comment:
How about calling this setAutosuggestValues? This would require you to get the autosuggestSelected items to add to current values, but it would be possible.

Or maybe a method with a callback would be cool, but I'm not sure about the naming:

```javascript

EasySearch.getComponentInstance(...).manageAutosuggestValues(function (current) {
  current.push({ id: 'test', value: 'test' });
});
```

This would add a new value, you could also modify the array of values to your needs. Might also make the resetAutosuggest obviate the resetAutosuggest.

wdyt?